CONTRIBUTING.md

```diff
@@ -170,11 +204,11 @@ To see what resources will be deployed for a live service, check the
 `test-resources.json` ARM template files in the service you wish to deploy for
 testing, for example `sdk\keyvault\test-resources.json`.
 
-To deploy live resources for testing use the steps documented in [`Example 1 of New-TestResources.ps1`](eng/common/TestResources/New-TestResources.ps1.md#example-1)
-to set up a service principal and deploy live testing resources.
+To deploy live resources for testing use the steps documented in [`Example 1 of New-TestResources.ps1`](eng/common/TestResources/New-TestResources.ps1.md#example-1) to set up a service principal and deploy live testing resources.
+
+The script will provide instructions for setting environment variables before running live tests.
 
-The script will provide instructions for setting environment variables before
-running live tests.
+Keep in mind that you can set the environment variables as how they're set by default through your system preferences, or through the command line by assigning values to these variable names one by one before running the tests, like `<name1>=<value1> <name2>=<value2> rushx <command>`, or by setting them each one in a different line in a `.env` file in the parent folder of the project you'll be testing (like in the folder `keyvault` if you're testing the project at `sdk/keyvault/keyvault-keys`).
```

Comment:
Among the different ways to set env variables here, the one using `.env` is the only one I would expect a program mer to not be aware of or to not be able to find by looking it up.

Reply:
Is this better?

> Keep in mind that you can set the environment variables as how they're set by default through your system preferences, or through the command line, or by setting them each one in a different line in a `.env` file in the parent folder of the project you'll be testing (like in the folder `keyvault` if you're testing the project at `sdk/keyvault/keyvault-keys`). You can read more about how `dotenv` works in [their README](https://github.com/motdotla/dotenv#readme).

Seems reasonable to mention the alternatives at least.

Reply:
There are a few concerns I have here:

- Starting a sentence with the phrase `Keep in mind` tells the reader that they need to follow a particular instruction otherwise would face unexpected consequences. What we are trying to convey here doesnt fall under that category.
- I would assume that most users would know how to set env variables in general. It is also something very easy to look up. I wouldnt want to give real estate to instructions that are not a result of specific decisions the testing infrastructure has taken. The use of `.env` file is definitely worth calling out, but others not so much. When we are showing a reader a big wall of text, it is important to keep it to the point and brief
- Say that you still want to show the alternatives, then I would recommend using a bulleted list so that it is easy to glance over, rather than read till the end of the paragraph to find the part that the user is not aware of

Reply:
> I would assume that most users would know how to set env variables in general. It is also something very easy to look up.

Even in our own team, we've had issues with environment variables. I believe it is better to reduce the assumptions, if possible.

Reply:
I think I will remove this paragraph